src/scripts/cfo.zig

```diff
@@ -336,12 +357,26 @@ fn run_fuzzers(
         }
     }
     assert(seeds.items.len == 0);
+
+    for (tasks_cache.?.seed_template, tasks_cache.?.runtimes) |template, runtime| {
+        log.debug("commit={s} fuzzer={s:<24} runtime={}", .{
+            template.commit_sha[0..7],
+            @tagName(template.fuzzer),
+            runtime,
+        });
+    }
 }
 
 const Tasks = struct {
+    /// Map value is the index within the task arrays.
+    const Map = std.AutoHashMap(struct { fuzzer: Fuzzer, commit: [40]u8 }, usize);
+
+    map: Map,
     working_directory: [][]const u8,
     seed_template: []SeedRecord.Template,
     weight: []u32,
+    /// Estimated cumulative runtime of the corresponding task.
+    runtimes: []u64,
 };
```

Comment:
It feels a bit weird that we do both an AoS and a Map here now. Perhaps its time to raise the level of abstraction here and do something like

```zig
const Tasks = struct {
    const Task = struct {
       ...
    };

    pub fn sample(tasks: *const Tasks) *Task {}
    pub fn get(tasks: *const Tasks, fuzzer: Fuzzer, commit: [40]u8) ?*Task {}
    pub fn get_or_insert(tasks: *const Tasks, task: *Task) *Task {}
    pub fn soft_remove_all() void {}
};
```

Though, I think the current code is fine, even if it was written with the idea that tasks are immutable.